doc/manual/parallel-computing.rst

```diff
@@ -858,10 +858,10 @@ Custom cluster managers would typically specify only ``io`` or ``host`` / ``port
 - ``count``, ``exename`` and ``exeflags`` are relevant for launching additional workers from a worker.
   For example, a cluster manager may launch a single worker per node, and use that to launch
   additional workers.
-    - ``count`` with an integer value ``n`` will launch a total of ``n`` workers.
-    - ``count`` with a value of ``:auto`` will launch as many workers as cores on that machine.
-    - ``exename`` is the name of the ``julia`` executable including the full path.
-    - ``exeflags`` should be set to the required command line arguments for new workers.
+  - ``count`` with an integer value ``n`` will launch a total of ``n`` workers.
```

Comment:
needs rebase because of https://github.com/JuliaLang/julia/pull/17761, sorry - the newline that @MichaelHatherly added before this nested list might be necessary